src/Tizen.Security.SecureRepository/Tizen.Security.SecureRepository/DataManager.cs

```diff
@@ -73,7 +73,7 @@ static public byte[] Get(string alias, string password)
         /// <since_tizen> 3 </since_tizen>
         /// <returns>All aliases of data, which the client can access.</returns>
         /// <exception cref="ArgumentException">No alias to get.</exception>
-        static public IEnumerable<string> GetAliases()
+        static public IReadOnlyCollection<string> GetAliases()
```

Comment:
Why do you change the return type?

Reply:
Even though the IEnumerable.Count() function will not enumerate all items, I believe it would be be beneficial to communicate to the caller that calling count will not do so. IReadOnlyCollection is literally IEnumerable<T> with the Count property, I can change it back if it not convincing enough.

Reply:
You can't change public API

Reply:
I would address the issue tomorrow. I would also just change the return type back to innumerable as the count() method is special cases for collection anyway so there is not much gain there